policy.go

```diff
@@ -5,205 +5,141 @@
 package kes
 
 import (
-	"encoding/json"
-	"errors"
-	"io"
+	"net/http"
+	"strings"
 	"time"
 )
 
-// Policy contains a set of rules that explicitly allow
-// or deny HTTP requests.
+// A Rule controls HTTP requests and is part of a policy.
+type Rule struct{}
```

Comment:
Do we have plans to add `Rule` attributes or some functions later?

Reply:
Yeah, in the future we will add options to tune access control